src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    final int MAX_FIGURE = 5;
+    final int DEFAULT_NUMBER = 10;
+    final String DEFAULT_COLOR = "WHITE";
```

Comment:
```suggestion
    private static final int MAX_FIGURE = 5;
    private static final int DEFAULT_RADIUS = 10;
    private staticfinal String DEFAULT_COLOR = "WHITE";
```

get WHITE from your Color enum